action/src/platforms/github.ts

```diff
@@ -76,9 +77,9 @@ export class GitHubPlatformKit extends PlatformKit {
     if (ghToken && processOwnCommits) {
       console.log("Using provided GH_TOKEN. This will trigger your CI/CD pipeline to run again.");
 
-      execSync(`git remote set-url origin https://${ghToken}@github.com/${repositoryOwner}/${repositoryName}.git`, {
-        stdio: "inherit",
-      });
+      const URL = `https://${ghToken}@github.com/${repositoryOwner}/${repositoryName}.git`;
```

Comment:
Let's use lowercase "url" for variables like this:

```suggestion
      const url = `https://${ghToken}@github.com/${repositoryOwner}/${repositoryName}.git`;
```